<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="tab_button_word">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>转为Word</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tab_button_excel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>转为Excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tab_button_ppt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>转为PPT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tab_button_split">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF拆分</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tab_button_merge">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF合并</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QWidget" name="word">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>转为word</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="excel">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>转为excel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ppt">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>转为ppt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="split">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF拆分</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="merge">
     <widget class="QListWidget" name="merge_listwidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>431</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_up_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>上移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_down_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>240</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>下移</string>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_delete_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_add_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>300</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF合并</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>输出名称：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="merge_output_field">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>373</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>.pdf</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>输出目录：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_folder_button">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>415</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF相同目录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="merge_start_button">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>415</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>开始转换</string>
      </property>
     </widget>
     <widget class="QLabel" name="merge_output_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>450</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>请选择目录</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>本软件获得勇哥大力支持</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
